Makefile

```diff
@@ -421,11 +421,15 @@ k8s_codegen:
 	@rm -rf $(K8S_CODE_GENERATOR)
 
 k8s_codegen_operator:
-	@tar zxvf $(K8S_CODE_GENERATOR).tar.gz -C ./../ --no-same-owner
+	@tar zxvf $(K8S_CODE_GENERATOR).tar.gz --no-same-owner
 	@chmod +x $(K8S_CODE_GENERATOR)/generate-groups.sh
 	@$(K8S_CODE_GENERATOR)/generate-groups.sh all $(OPERATOR_KUBERNETES_PKG)/client \
-		$(OPERATOR_KUBERNETES_PKG)/apis "netapp:v1" -h ./../hack/boilerplate.go.txt
+		$(OPERATOR_KUBERNETES_PKG)/apis "netapp:v1" -h ./hack/boilerplate.go.txt
 	@rm -rf $(K8S_CODE_GENERATOR)
+	@rm -rf ./operator/controllers/orchestrator/client/*
```

Comment:
Curious why the Makefile changes are needed.  Is this stanza somehow broken?

Reply:
Yes, you could run `make k8s_codegen_operator` on your local machine without this change and it will fail.